README.md

```diff
@@ -22,72 +20,138 @@ Then as usual, run:
 bundle install
 ```
 
+## Usage
+
+Run `$middleman contentful` in your terminal. This will fetch entries for the configured
```

Comment:
Changed in https://github.com/madtrick/contentful_middleman/commit/0317577bb553e95291640fcb26b9c42470578a1f